docs/.templates/integration/schema.json

```diff
@@ -0,0 +1,84 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories_list": {
+            "type": "object",
+            "description": "A list defining all the available categories for the integrations",
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of the category"
+                            },
+                            "priority": {
+                                "type": "string",
+                                "description": "Priority of the category"
```

Comment:
What is `priority`? How does it work? Can you add more info to its description?

Reply:
John explained it [here](https://github.com/netdata/netdata/pull/15302/files#r1250981042), but I will do my best to try and explain it on the schema